package-lock.json

```diff
@@ -5,9 +5,9 @@
   "requires": true,
```

Comment:
There is a whole lot of changes to the package lock here..... are these needed for everyone?

Reply:
Added package lock to ignore